packages/admin/blocks-admin/src/blocks/factories/createColumnsBlock.tsx

```diff
@@ -246,9 +246,10 @@ export function createColumnsBlock<T extends BlockInterface>({
                                     component={FinalFormLayoutSelect}
                                     layouts={groupLayoutsByColumnsApi.getLayouts(state.layout.columns)}
                                     fullWidth
+                                    numberOfMatchingColumns={layouts.filter((layout) => layout.columns === state.layout.columns).length}
```

Comment:
Please use `layouts` for this, which is already  filtered by the number of columns

Reply:
Please remove the prop here

Reply:
```suggestion
```